include/wlr_screencast.h

```diff
@@ -12,7 +12,8 @@
 
 #define XDG_OUTPUT_MANAGER_VERSION 3
 
-#define LINUX_DMABUF_VERSION 3
+#define LINUX_DMABUF_VERSION 4
```

Comment:
Nit: maybe rename to `_MIN`?

Reply:
How should wayland protocol versioning work? My guess would be to aim at the latest available version, which has usefull features for us and then add a fallback if possible for lower versions. With this semantics we have
- (Aimed) Version
- Minimum version